src/main/java/org/opentripplanner/common/MavenVersion.java

```diff
@@ -111,6 +113,30 @@ public String toStringVerbose() {
         return String.format("%s => %s UID=%d", version, this.toString(), getUID());
     }
 
+    public String getVersion(String fragment) {
```

Comment:
I was in doubt about using an `enum` for this. If anyone objects to using a `String` here, I will happily create an `enum`.